src/platform/user/profile/vet360/components/AddressField/AddressEditModal.jsx

```diff
@@ -7,6 +7,11 @@ import Vet360EditModal from '../base/Vet360EditModal';
 
 import CopyMailingAddress from 'vet360/containers/CopyMailingAddress';
 import AddressForm from './AddressForm';
+import AddressFormV2 from './AddressFormV2';
+
+import environment from 'platform/utilities/environment';
+
+const useNewAddressForm = environment.isLocalhost();
```

Comment:
Right now we only want to use this new address form locally as we are about to start manual QA on a different feature that uses the existing form and we don't want to create failures during QA that have nothing to do with the feature they are really trying to test.